versions.sh

```diff
@@ -0,0 +1,15 @@
+ipxe_bluebanquise_version=1.1.3
```

Comment:
I would rename the file `versions` or `versions.conf`

Reply:
done